packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1321,6 +1331,14 @@ class SemanticsProperties extends DiagnosticableTree {
   /// [Directionality] or an explicit [textDirection] should be provided.
   final String? tooltip;
 
+  /// The level of the widget (when it's defined as a header) within
```

Comment:
same

Reply:
Also changed this, hope it's better now. Thank you.